update_output_handler.py

```diff
@@ -0,0 +1,33 @@
+try:
+    import RPi.GPIO as GPIO
+except ImportError:
+    print("RPi.GPIO not found, using simulated version.")
+    try:
+        import SimulRPi.GPIO as GPIO
+    except ImportError:
+        print("SimulRPi not found, exiting.")
+        exit(1)
```

Comment:
To możesz raczej wynieść do osobnego pliku dla zachowania klarowności, i wtedy wystarczy import from z tego pliku